test/common.js

```diff
@@ -21,6 +76,8 @@ if (process.env.TEST_THREAD_ID) {
 }
 exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
+refreshTmpDir();
```

Comment:
The current Python script deletes and creates the temporary directory for each and every test. This duplicates that functionality. Making it something the individual tests call might be a decent performance win but it would also be a more invasive change, touching a lot of tests. I'd rather do that as two separate PRs.